src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js

```diff
@@ -10,8 +10,20 @@ describe('Personal health care contacts -- feature enabled', () => {
     cy.intercept('GET', '/v0/profile/contacts', contacts);
     cy.login(loa3User72);
     cy.visit(PROFILE_PATHS.CONTACTS);
-    cy.findByTestId('phcc-emergency-contact');
-    cy.findByTestId('phcc-next-of-kin');
+    cy.findByTestId('phcc-emergency-contact-0');
+    cy.findByTestId('phcc-emergency-contact-1');
+    cy.findByTestId('phcc-next-of-kin-0');
+    cy.findByTestId('phcc-next-of-kin-1');
+    cy.injectAxeThenAxeCheck();
+  });
+
+  it('displays instructions when no contacts are present', () => {
+    const featureToggles = generateFeatureToggles({ profileContacts: true });
+    cy.intercept('GET', '/v0/feature_toggles*', featureToggles);
+    cy.intercept('GET', '/v0/profile/contacts', { data: [] });
+    cy.login(loa3User72);
+    cy.visit(PROFILE_PATHS.CONTACTS);
+    cy.get(/To add a contact, call us/);
```

Comment:
Can you use findByRole here?

Reply:
The Instructions component can be rendered multiple times in the same page. `cy.find*` calls fail if more than one match is found. `cy.get` calls can handle multiple matches.